api/app/signals/apps/questionnaires/services/feedback_request.py

```diff
@@ -90,62 +91,68 @@ def create_kto_graph():
     return graph
 
 
-class FeedbackRequestService:
-    @staticmethod
-    def get_feedback_urls(session):
-        fe_location = get_fe_application_location()
-        positive_feedback_url = f'{fe_location}/feedback/ja/{session.uuid}'
-        negative_feedback_url = f'{fe_location}/feedback/nee/{session.uuid}'
+def get_feedback_urls(session):
+    fe_location = get_fe_application_location()
+    positive_feedback_url = f'{fe_location}/feedback/ja/{session.uuid}'
+    negative_feedback_url = f'{fe_location}/feedback/nee/{session.uuid}'
 
-        return positive_feedback_url, negative_feedback_url
+    return positive_feedback_url, negative_feedback_url
 
-    @staticmethod
-    def create_session(signal):
-        if signal.status.state != workflow.AFGEHANDELD:
-            msg = f'Signal {signal.id} is not in state REACTIE_GEVRAAGD!'
-            raise WrongState(msg)
 
-        with transaction.atomic():
-            graph = create_kto_graph()
-            questionnaire = Questionnaire.objects.create(graph=graph, flow=Questionnaire.FEEDBACK_REQUEST)
-            session = Session.objects.create(
-                submit_before=timezone.now() + timedelta(days=FEEDBACK_REQUEST_DAYS_OPEN),
-                questionnaire=questionnaire,
-                _signal=signal,
-            )
-
-        return session
-
-    @staticmethod
-    def handle_frozen_session_FEEDBACK_REQUEST(session):
-        from signals.apps.questionnaires.services import QuestionnairesService
-
-        if not session.frozen:
-            msg = f'Session {session.uuid} is not frozen!'
-            raise SessionNotFrozen(msg)
-        if session.questionnaire.flow != Questionnaire.FEEDBACK_REQUEST:
-            msg = f'Questionnaire flow property for session {session.uuid} is not FEEDBACK_REQUEST!'
+def create_session_for_feedback_request(signal):
+    if signal.status.state != workflow.AFGEHANDELD:
```

Comment:
This check does not matches the msg raised

Reply:
fixed